training/utils.py

```diff
@@ -8,7 +8,8 @@
 
 CHECKPOINT_SIZE_MB = 333
 BATCH_SIZE_PER_GB = 2.5
-LEARNING_RATE_PER_BATCH = 3.125e-5
+LEARNING_RATE_PER_64 = 4e-4 # corrected learning rate to match paper. # Tacotron2 alignment forms best between 2e-4 and 5e-5
```

Comment:
Please remove second # in comment

Reply:
little busy, do it for me if you can, overwise wait for me